### ISSUE TITLE
Issue with /docs/tasks/access-application-cluster/port-forward-access-application-cluster/

### ISSUE THREAD

**Issue by mshiriv** [kubernetes/website#28994]
According to doc when applying manifests, get `404 Page not found`:
```
$ kubectl apply -f https://k8s.io/examples/application/guestbook/mongo-deployment.yaml
error: unable to read URL "https://k8s.io/examples/application/guestbook/mongo-deployment.yaml", server reported 404 Not Found, status code=404
```
```
$ kubectl apply -f https://k8s.io/examples/application/guestbook/mongo-service.yaml
error: unable to read URL "https://k8s.io/examples/application/guestbook/mongo-service.yaml", server reported 404 Not Found, status code=404
```
Seems the manifests URL change did not apply to the doc.

**Comment by jihoon-seo** [comment]
This issue will be resolved when the PR Add MongoDB deployment and service. #28530 is merged.

(Use Port Forwarding to Access Applications in a Cluster #28823)

- (#28854)

**Comment by tylerauerbeck** [comment]
@jihoon-seo I see that #28530 has been merged and it appears that this still results in a 404 error. Is this expected?

**Comment by jihoon-seo** [comment]
@tylerauerbeck Please note that the URLs of YAML files are slightly changed.

In https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/ ,
there are 2 YAML URLs.

- https://k8s.io/examples/application/mongodb/mongo-deployment.yaml
- https://k8s.io/examples/application/mongodb/mongo-service.yaml

And with my web browser, I can reach those 2 YAML files.

Please check again, and if the problem still exists, then let me know by writing a comment here!

---

This issue is resolved since the PR #28530 has been merged.
